projects/canopy/src/lib/details/details.component.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../styles/mixins';
+@import '../../styles/variants.scss';
```

Comment:
It's rare that I condone comments, but I think this could do with one. Without knowing the reasoning it looks like this is a redundant import (does it need the file extension?).

Do we include it for now but tag it as an eventual breaking change? With the view to bringing in that import another way? (finding a nice way might be tricky)

Reply:
Sure, I can add a comment and remove the `scss` extension.
My plan is once this gets merged i'll rebase `next` and revert these changes so that it can go in as a breaking change with all the rest.